samples/sample_encode/include/pipeline_encode.h

```diff
@@ -169,6 +171,7 @@ struct sInputParams
     bool shouldUseShiftedP010Enc;
     bool shouldUseShiftedP010VPP;
 
+    msdk_char *RoundingOffsetFile;
```

Comment:
please add default initialization to this pointer

Reply:
OK